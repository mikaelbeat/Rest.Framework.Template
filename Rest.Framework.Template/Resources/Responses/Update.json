{"name":"morpheus","job":"zion resident","updatedAt":"2018-06-12T11:17:44.624Z"}