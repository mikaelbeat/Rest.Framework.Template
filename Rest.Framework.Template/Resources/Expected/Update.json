{"name":"morpheus","job":"zion resident","updatedAt":"2018-06-12T10:45:17.380Z"}